Resources/colyseus-unity-sdk-upm/Runtime/GameDevWare.Serialization/Metadata/MetadataReflection.cs

```diff
@@ -204,24 +202,25 @@ public static bool TryGetConstructor(Type type, out Func<object> ctrFn, out Cons
 
 			defaultConstructor = type.GetConstructors(BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic).FirstOrDefault(ctr => ctr.GetParameters().Length == 0);
 
+			if (AotRuntime && defaultConstructor != null) {
```

Comment:
really minor thing, can we keep opening and closing brackets on separate lines

Reply:
I agree, though as it's a third-party dependency I'd be keen to keep it the same as their copy